src/tcpconns.c

```diff
@@ -171,6 +176,24 @@ struct tcpi_field_selector {
 static struct tcpi_field_selector *tcpi_fields_to_report = NULL;
 static size_t num_tcpi_fields_to_report = 0;
 static size_t tcpi_fields_to_report_size = 0;
+static size_t num_tcpi_counter_fields = 0;
+
+static uint64_t counter_max_delta = 1000000;
+
+/* Cache for computing deltas of DELTA values. */
+static cdtime_t counter_cache_timeout = DOUBLE_TO_CDTIME_T(120.0);
```

Comment:
That might be problematic. We're hoping to get a kernel-side filtering mechanism to keep quiet sockets out of the netlink report. But I won't want to lose them. Do you think we should pay the extra programming cost now and see about fixing it later?